modules/geo-layers/src/tile-layer/tile-2d-header.ts

```diff
@@ -13,14 +40,16 @@ export default class Tile2DHeader {
 
     this.content = null;
 
+    this._loader = undefined;
+    this._abortController = null;
     this._loaderId = 0;
     this._isLoaded = false;
     this._isCancelled = false;
     this._needsReload = false;
   }
 
   get data() {
```

Comment:
Forward layer data type `DataT` to `Tile2DHeader<DataT>`?

Reply:
In light of https://github.com/visgl/deck.gl/pull/6847 not yet being concluded I think it is best if we leave this for now and tackle in a new PR to enable us to close this PR

Reply:
I was hesitating if this is exactly the same type and i was think _they_ may not be the same
Apparently, at least for MVTLayer they are the same. Not sure if this is universally true, so i concur with @felixpalmer suggestion to prepare separate patch and work on this issue separatedly.

Reply:
Promised attempt to make `Tile2DHeader` generic: #6889